src/features/prompts/server/prompt-router.ts

```diff
@@ -27,18 +27,44 @@ export const promptRouter = createTRPCRouter({
         projectId: input.projectId,
         scope: "prompts:read",
       });
-      return ctx.prisma.prompt.findMany({
+      const prompts = await ctx.prisma.$queryRaw<Array<Prompt>>`
```

Comment:
The typing here is dangerous. Typescript types this to Prompt but we did not select all the db columns (e.g. project_id, updated_at)